tutorials/fundamentals/check-integer.md

```diff
@@ -0,0 +1,28 @@
+To check if a variable is an integer in JavaScript, use `Number.isInteger()`.
+`Number.isInteger()` returns `true` or `false` depending on the parameter provided.
+
+```javascript
+let example = 12.1;
+Number.isInteger(example); // false
+example = 12;
+Number.isInteger(example); // true
+example = Infinity;
+Number.isInteger(example); // false
+```
+The neat thing about `Number.isInteger()` is that you can pass mathmatical expressions which the function will evaluate and return the boolean result.
+
+```javascript
+let example = 4/2;
+Number.isInteger(example); // true
+example = 3/2;
+Number.isInteger(example); // false
+```
+
+A quirk that `Number.isInteger()` has is that it will round after 15 decimal places.
```

Comment:
Replace this with "Remember that JavaScript can only represent up to [16 decimal places](https://stackoverflow.com/questions/54800022/why-max-digits-with-decimal-in-javascript-are-only-16), so `Number.isInteger()` may return surprising results in cases where JavaScript doesn't have sufficient numeric precision to represent the output."